src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/Blocks/View/products.html.twig

```diff
@@ -0,0 +1,134 @@
+  </h3>
+  <div class="card-body">
+    {% for product in viewableManufacturer.manufacturerProducts %}
+      <div class="card">
+        <div class="card-header clearfix">
+          <a href="{{ path('admin_product_form', {'id': product.id}) }}">{{ product.name }}</a>
+
+          <div class="d-inline-block float-right">
+            <div class="btn-group-action text-right">
+              <div class="btn-group">
+                <a class="btn btn-link dropdown-toggle dropdown-toggle-dots dropdown-toggle-split p-0" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                </a>
+                <div class="dropdown-menu dropdown-menu-right">
+                  <a class="btn tooltip-link js-submit-row-action dropdown-item"
+                     href="{{ path('admin_product_form', {'id': product.id}) }}"
+                  >
+                    {{ 'Edit'|trans({}, 'Admin.Actions') }}
+                  </a>
+                  <button class="btn tooltip-link js-form-submit-btn dropdown-item"
+                          type="button"
+                          data-form-submit-url="{{ path('admin_product_unit_action', {'action': 'delete', 'id': product.id}) }}"
+                          data-form-confirm-message="{{ '%s%s?'|format('Delete item #'|trans({}, 'Admin.International.Feature'), product.id) }}"
+                  >
+                    {{ 'Delete'|trans({}, 'Admin.Actions') }}
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="card-body">
+          {% if product.combinations is not empty %}
+            <table class="table">
+              <thead>
+                <tr>
+                  <th>{{ 'Attribute name'|trans({}, 'Admin.Catalog.Feature') }}</th>
+                  <th>{{ 'Reference'|trans({}, 'Admin.Global') }}</th>
+                  <th>{{ 'EAN13'|trans({}, 'Admin.Catalog.Feature') }}</th>
+                  <th>{{ 'UPC'|trans({}, 'Admin.Catalog.Feature') }}</th>
+                  {% if not isAllShopContext and isStockManagementEnabled %}
+                    <th>{{ 'Available quantity'|trans({}, 'Admin.Catalog.Feature') }}</th>
+                  {% endif %}
+                </tr>
+              </thead>
+              <tbody>
+                {% for combination in product.combinations %}
+                  <tr>
+                    <td>{{ combination.attributes }}</td>
+                    <td>{{ combination.reference }}</td>
+                    <td>{{ combination.ean13 }}</td>
+                    <td>{{ combination.upc }}</td>
+                    {% if not isAllShopContext and isStockManagementEnabled %}
+                      <td>{{ combination.quantity }}</td>
+                    {% endif %}
+                  </tr>
+                {% endfor %}
+              </tbody>
+            </table>
+          {% else %}
+            <table class="table">
+              <thead>
+                <tr>
```

Comment:
I don't it this was discussed with Tristan or someone for UI, but it seems to me it would be easier to read if the tables always have the same columns (even if they are empty).
Here from one product to another you can have from 1 to 4 columns, with different sizes each time.

Reply:
Agree with that. :+1:

Reply:
so what do we do about it? :thinking:

Reply:
I think you can remove the `if` conditions

Reply:
done. :+1:

Reply:
thanks